api/src/server.rs

```diff
@@ -409,15 +410,24 @@ where
         ))
     }
 
+    // TODO(stbrody): Only take the event data, don't take an id field at all since its unused.
     pub async fn post_events(&self, event: Event) -> Result<EventsPostResponse, ErrorResponse> {
```

Comment:
we might need to change this endpoint to accept something else (e.g. EventInput) that is just a data field or make eventID optional, but that would be weird for the get endpoints.

Reply:
yep, probably going to need to introduce a new type.

I'm going to do that change in a separate PR as a follow-up